src/index.js

```diff
@@ -92,13 +92,20 @@ class CID {
     return this.buffer
   }
 
+  toV0String () {
+    if (this.version === 0) {
+      return mh.toB58String(this.multihash)
+    }
+
+    throw new Error(`Not supported on version: ${this.version}`)
+  }
+
   /* defaults to base58btc */
   toBaseEncodedString (base) {
     base = base || 'base58btc'
 
     switch (this.version) {
       case 0:
-        return mh.toB58String(this.multihash)
       case 1:
```

Comment:
Maybe this could be solved with some more straight forward naming.

```js
cidV0.toString() 
// => base58
cidV1.toString()
// => base58 with multibase prefix

cidv0.toMultibaseString('base16')
// base16 with multibase prefix
cidv0.toMultibaseString('base58btc')
// base58 with multibase prefix

cidv1.toMultibaseString('base16')
// base16 with multibase prefix
cidv1.toMultibaseString('base58btc')
// base58 with multibase prefix

cidV0.toMultibaseString()
// throw Error, required base

cidV1.toMultibaseString()
// throw Error, required base
```